packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts

```diff
@@ -12,6 +12,7 @@ import menuButtonBehavior from '../MenuButton/menuButtonBehavior'
 const splitButtonBehavior: Accessibility = props => {
   const splitButtonMenuButtonBehavior = props => {
     const menuButtonBehaviorData = menuButtonBehavior(props)
+    menuButtonBehaviorData.attributes.trigger['aria-haspopup'] = undefined
```

Comment:
Shouldn't we add more info in the @specification?

Reply:
as aria-popup shouldn't be add I am not writing to the description, that we are not adding it, but add the test to verify it :)